Source/Blazorise/Base/BaseTextInput.razor.cs

```diff
@@ -143,6 +143,7 @@ protected override Task OnBlurHandler( FocusEventArgs eventArgs )
         private void OnInputValueDebounce( object sender, string value )
         {
             InvokeAsync( () => CurrentValueHandler( value ) );
+            InvokeAsync( StateHasChanged );
```

Comment:
Hmm, is StateHasChanged needed here?

Reply:
I'm testing without `InvokeAsync( StateHasChanged );` on my side and it seems to work just fine. Can you also test and conform yourself?

Reply:
If you remove it, it goes back to happening as it goes out of sync. Simply type a bunch of words as if you were actually writting some paragraph and try to do it on a slightly more paused manner.

For example with this
`<TextEdit Debounce="true" DebounceInterval="300"  />`

So I would ask you to retest on your end with and without.

Reply:
Working with StateHasChanged